app/views/shared/_friend_control.html.erb

```diff
@@ -0,0 +1,15 @@
+<span class="add-<%= user.id %>">
+    <% if user != current_user %>
+        <% if current_user.friendship?(user) %>
```

Comment:
[OPTIONAL]

It's a good idea to indent your ruby logic. Please use this same approach to lines 7, 9 and others in similar context within this entire repo.